messages/en.json

```diff
@@ -1,25 +1,36 @@
 {
-    "LoginPage": {
-        "LogIn": "Log in",
-        "DONT_HAVE_ACCOUNT": "You don't have an account?",
-        "SIGN_UP":"Sign up"
-    },
+  "LoginPage": {
+    "LogIn": "Log in",
+    "DONT_HAVE_ACCOUNT": "You don't have an account?",
+    "SIGN_UP": "Sign up"
+  },
 
-    "LanguageSelector": {
-        "locale": "{locale, select, es {Spanish} en {English} other {Unknown}}",
-        "Language": "Language"
-    },
+  "LanguageSelector": {
+    "locale": "{locale, select, es {Spanish} en {English} other {Unknown}}",
+    "Language": "Language"
+  },
 
-    "RegisterPage": {
-        "NAME": "Name",
-        "USER_NAME": "User name",
-        "NAME_TO_RECOGNIZE": "Name you will be recognized by",
-        "EMAIL": "Email",
-        "PASSWORD": "Password",
-        "CONFIRM_PASSWORD": "Confirm password",
-        "ACCEPT_TERMS_AND_CONDITIONS": "Accept terms and conditions",
-        "REGISTER": "Register",
-        "ALREADY_REGISTERED": "Do you already have an account?",
-        "LOG_IN": "Log in"
-    }
-}
\ No newline at end of file
+  "RegisterPage": {
+    "NAME": "Name",
+    "USER_NAME": "User name",
+    "NAME_TO_RECOGNIZE": "Name you will be recognized by",
+    "EMAIL": "Email",
+    "PASSWORD": "Password",
+    "CONFIRM_PASSWORD": "Confirm password",
+    "ACCEPT_TERMS_AND_CONDITIONS": "Accept terms and conditions",
+    "REGISTER": "Register",
+    "ALREADY_REGISTERED": "Do you already have an account?",
+    "LOG_IN": "Log in"
+  },
+
+  "Validation": {
```

Comment:
change to `ValidationRegisterPage `because in the future there will be other validations. so is a good practice identify what type of validation belong.

Reply:
Done